src/main/java/me/maxouxax/multi4j/MultiHelper.java

```diff
@@ -64,4 +69,56 @@ public static ArrayList<UnivClass> getUnivScheduleForUser(MultiClient multiClien
         return univClasses;
     }
 
+    public static RestoCrous getMenuRu(MultiClient mc, String nomRu) throws MultiLoginException, IOException, URISyntaxException, InterruptedException {
+        mc.connect();
+
+        JSONObject jsonObject = new JSONObject("{\"operationName\":\"crous\",\"variables\":{},\"query\":\"query crous {\\n  restos {\\n    title\\n    thumbnail_url\\n    image_url\\n    short_desc\\n    lat\\n    lon\\n    menus {\\n      date\\n      meal {\\n        name\\n        foodcategory {\\n          name\\n          dishes {\\n            name\\n            __typename\\n          }\\n          __typename\\n        }\\n        __typename\\n      }\\n      __typename\\n    }\\n    __typename\\n  }\\n}\\n\"}");
+        JSONObject response = mc.makeGQLRequest(jsonObject);
+
+        SimpleDateFormat s = new SimpleDateFormat("yyyy-MM-dd");
+        Date aujd = new Date();
+        String date = s.format(aujd);
+        int findCrous = 0;
+
+        //Chercher Crous
+        JSONArray recherche = response.getJSONObject("data").getJSONArray("restos");
+        for (int i = 0; i < recherche.length(); i++) {
+            if (recherche.getJSONObject(i).getString("title").equals(nomRu)) {
+                findCrous = i;
+                break;
+            }
+        }
+
+        JSONObject suite = response.getJSONObject("data").getJSONArray("restos").getJSONObject(findCrous);
+        String titleRu = suite.get("title").toString();
+        String imageRu = suite.get("thumbnail_url").toString();
+        JSONArray tjrlarecherche = suite.getJSONArray("menus");
+        int findDate = 0;
+        String dateRu= "";
+        
+        //Chercher aujd
+        for (int i = 0; i < tjrlarecherche.length(); i++) {
+            if (tjrlarecherche.getJSONObject(i).getString("date").equals(date)) {
+                dateRu = tjrlarecherche.getJSONObject(i).getString("date");
+                findDate = i;
+                break;
+            }
+        }
+        
+        JSONObject encoreouais = suite.getJSONArray("menus").getJSONObject(findDate);
+
+        //Recuperer Repas
+        JSONObject tjrtjr = encoreouais.getJSONArray("meal").getJSONObject(0);
+        JSONObject tjrtjrr = tjrtjr.getJSONArray("foodcategory").getJSONObject(0);
+        JSONArray tjrtjrrarray = tjrtjrr.getJSONArray("dishes");
+        String sli = "";
+        for (int i = 0; i < tjrtjrrarray.length(); i++) {
+            String temp = tjrtjrrarray.getJSONObject(i).getString("name");
+            sli += temp+" - ";
+        }
+        String menuRu = sli.substring(0, sli.length()-2);
+        
+        return (new RestoCrous(titleRu, imageRu, dateRu, menuRu));
```

Comment:
It would be nice if you could rename those variables to what they are actually used for - giving weird names like that isn't great :(